azure-pipelines.yml

```diff
@@ -91,15 +91,15 @@ stages:
         displayName: UT FT common & flink & UT client/spark-client
         timeoutInMinutes: '150'
         steps:
-          - task: Maven@3
+          - task: Maven@3.205.1
```

Comment:
is this to silence the warning? what about the actual fix, i suppose the warnings are valid improvement suggestions?

Reply:
refer to https://github.com/microsoft/azure-pipelines-tasks/issues/16520, 
This pinpoints the issue caused by a change in the Maven task from version 3.205.1 to 3.206.0.
.  they will fix this in the future.  but using 3.205.1 can be a temporary workaround

Reply:
@fengjian428 my question is: no matter which maven version used, are the currently surfaced warnings valid? don't we need to fix from the code?

Reply:
there are code compilation warnings too, not just the "replace images source" one

Reply:
then i think this is ok

Reply:
there are two warnings, one is caused by azure-pipelines itself, and I feel no need to be fixed. another is caused by we use sun.util.calender.ZoneInfo, maybe can replace with [java.time.zone.ZoneRules](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/time/zone/ZoneRules.html), Let me try it

Reply:
Already use ZoneId to replace sun.util.calendar.ZoneInfo